api/src/identity-access-management/infrastructure/repositories/last-user-application-connections.repository.js

```diff
@@ -8,4 +9,19 @@ async function upsert({ userId, application, lastLoggedAt }) {
     .merge({ lastLoggedAt });
 }
 
-export const lastUserApplicationConnectionsRepository = { upsert };
+async function findByUserId(userId) {
+  const lastUserApplicationConnections = await knex(TABLE_NAME).where({ userId }).orderBy('id', 'ASC');
+
+  return lastUserApplicationConnections.map(_toDomain);
+}
```

Comment:
```suggestion
async function findByUserId(userId) {
  const knexConn = DomainTransaction.getConnection();
  const lastUserApplicationConnections = await knexConn(TABLE_NAME).where({ userId });

  return lastUserApplicationConnections.map(_toDomain);
}
```

- À mettre également sur le `upsert` du repository.
- Supprimer le orderby car pas nécessaire dans la PR.